createdisk-library.sh

```diff
@@ -76,36 +76,13 @@ function update_json_description {
     local srcDir=$1
     local destDir=$2
 
-    diskSize=$(du -b $destDir/${CRC_VM_NAME}.qcow2 | awk '{print $1}')
-    diskSha256Sum=$(sha256sum $destDir/${CRC_VM_NAME}.qcow2 | awk '{print $1}')
-
-    ocSize=$(du -b $destDir/oc | awk '{print $1}')
-    ocSha256Sum=$(sha256sum $destDir/oc | awk '{print $1}')
-
-    podmanSize=$(du -b $destDir/podman-remote | awk '{print $1}')
-    podmanSha256Sum=$(sha256sum $destDir/podman-remote | awk '{print $1}')
+    crc-bundle-generator node "$destDir" --internal-ip ${INTERNAL_IP} --hostname "${VM_PREFIX}-master-0"
+    crc-bundle-generator cluster-info "$destDir"  --name "${destDir}" --kubeconfig-path "kubeconfig" --ssh-key "id_ecdsa_crc"
+    crc-bundle-generator add "$destDir" "$destDir/${CRC_VM_NAME}.qcow2" --type disk-image --format qcow2
+    crc-bundle-generator add "$destDir" "$destDir/oc" --type oc
+    crc-bundle-generator add "$destDir" "$destDir/podman-remote" --type podman
```

Comment:
Where is this `crc-bundle-generator` function ?

Reply:
ah this part is wip, was not supposed to be pushed.